docs/extended-tour.md

```diff
@@ -172,6 +173,40 @@ User of your module runs 'npm install'
 
 This binary file `../lib/binding/module.node` is what `require()` points to within Node.js.
 
+# Clang Tools
+
+This skeleton uses two clang llvm tools for automated formatting and static fixes. 
+
+Each of these tools run within their own [Travis jobs](https://github.com/mapbox/node-cpp-skel/blob/master/.travis.yml). You can disable these Travis jobs if you'd like, by commenting them out in `.travis.yml`. This may be necessary if you're porting this skel into an already-existing project that triggers tons of clang-tidy errors, and you'd like to work through them gradually while still actively iterating on other parts of your code.
+
+### [clang-format](https://clang.llvm.org/docs/ClangFormat.html)
+Automates coding style enforcement, for example whitespace, tabbing, wrapping.
+
+To run
+```
+make format
+```
+
+- Set format preferences in [`.clang-format`](https://github.com/mapbox/node-cpp-skel/blob/master/.clang-format)
+- Installed and run via [`/scripts/clang-format.sh`](https://github.com/mapbox/node-cpp-skel/blob/master/scripts/clang-format.sh)
+
+
+### [clang-tidy](https://clang.llvm.org/extra/clang-tidy/)
+Lint framework that can do very powerful checks, for example bugs that can be deduced via [static analysis](https://github.com/mapbox/cpp/blob/master/glossary.md#static-analysis), unneeded copies, inefficient for-loops, and improved readability. In many cases, it can fix and modernize your code.
+
+To run
+```
+make tidy
+```
+
+- If clang-tidy outputs a failure, it will also output a specific code. Check llvm's clang docs for that exact code to learn more. For example, see ["performance-unnecessary-copy-initialization"](https://clang.llvm.org/extra/clang-tidy/checks/performance-unnecessary-copy-initialization.html).
+- Set tidy preferences in [`.clang-tidy`](https://github.com/mapbox/node-cpp-skel/blob/master/.clang-tidy). When you run clang-tidy, you'll see output of a number of specific checks that tidy runs. You can add/remove any of these checks via the `.clang-tidy` file, configure these checks, and specify the files you'd like to run these checks against. The skel runs clang-tidy against everything within the `/src` directory.
```

Comment:
`cant` -> `can't`

Reply:
Whoops, this was meant to refer to the line below that reads `you either want to keep as-is or cant control`